shared/src/main/java/uk/gov/di/authentication/shared/validation/AuthAppCodeValidator.java

```diff
@@ -96,6 +96,10 @@ public Optional<String> getMfaCredentialValue() {
     }
 
     public boolean isCodeValid(String code, String secret) {
+        if (code.isEmpty() || code.length() > 9) {
```

Comment:
sticking to 6 digits? so could check > 6 ?

Reply:
thanks - changed